### DISCUSSION TITLE
The include-templates flag is not taken into account

### DISCUSSION THREAD

**Discussion by Techbrunch** [projectdiscovery/nuclei#1748] [Q&A] [answered]
I'm using the following .nuclei-ignore:

```
  GNU nano 2.0.6 File: ...loha/.config/nuclei/.nuclei-ignore

# ==| Nuclei Templates Ignore list |==
# ====================================
#
# This is default list of tags and files to excluded from default nucl$
# More details - https://nuclei.projectdiscovery.io/nuclei/get-started$

# tags is a list of tags to ignore execution for
# unless asked for by the user.

tags:
  - "fuzz"
  - "dos"

# files is a list of files to ignore template execution
# unless asked for by the user.

files:
  - cves/2020/CVE-2020-35489.yaml
```

I have the following entry in .config/nuclei/config.yaml

```
# templates to be executed even if they are excluded either by default$
include-templates:
  - fuzzing/iis-shortname.yaml
```

but when I run nuclei the template is not executed so it looks like that the template is not run because of the fuzz tag being ignored but the comment in the config says it should.

**Comment by ehsandeep** [comment]
@Techbrunch templates can be excluded in `.nuclei-ignore` using **tags** or **template** path and to overwrite we need to use the opposite flag, i.e `include-tags` for tags and `include-templates` for templates, since `fuzzing/iis-shortname.yaml` is excluded using `fuzz` tags, it can be included using `include-tags` only, so you need to use `include-tags` in the config file instead `include-templates`:

```
include-tags:
  - fuzz
```

**Reply by Techbrunch** [reply]
@ehsandeep  Thanks for the info but to me it sounds like this issue should have been converted to a feature request since I don't want to enable all fuzz templates but only this one. The comment is also confusing since it says "templates to be executed even if they are excluded either by default".

**Reply by ehsandeep** [reply]
@Techbrunch this is now supported in latest version of the project.

**Comment by forgedhallpass** [comment, marked as answer]
@Techbrunch you are right, I've created an issue for this: https://github.com/projectdiscovery/nuclei/issues/2124

**Comment by ehsandeep** [comment]
@Techbrunch this should be now supported in dev version of nuclei with https://github.com/projectdiscovery/nuclei/pull/2232
